overkilling/overkill-todo-monolith-api
# Add prometheus instrumentation

### Description

Add metrics gathering for HTTP handlers and repository requests.

### How to test

Steps to test :
1. Startup application
2. Make a few requests to `/todos` endpoint
3. Check `/metrics` for some relevant metrics
